### PR TITLE
Docs : Better quick start

### PR DESCRIPTION
**Pull request by Kayanski**
Add a better Quick Start documentation. 
This should be focused on faster integration

Closes ORC-39
Closes ORC-41 partially

Includes a broken link that will be solved by #240 (starship in the intro)

### CODE REVIEW

File: packages/cw-orch-contract-derive/src/lib.rs [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
 
-mod cw_orch_contract;
-
-use crate::cw_orch_contract::{
-    get_crate_to_struct, get_func_type, get_response_generic_or_fallback, get_wasm_name,
-};
-
-use convert_case::{Case, Casing};
-use syn::{__private::TokenStream2, parse_macro_input, Fields, FnArg, GenericArgument, Item, Path};
+use syn::{__private::TokenStream2, parse_macro_input, Fields, GenericArgument, Item, Path};
```
Comment (CyberHoward) [inline packages/cw-orch-contract-derive/src/lib.rs:3]: Hmmm should we be using this? `__private::TokenStream2`

Comment (Kayanski) [reply packages/cw-orch-contract-derive/src/lib.rs:3]: Yeah I know, but not sure how to do better

**Comment by Kayanski** [comment]
> This includes a lot more than just docs changes it looks like!

Yeah, mostly doc comments and code changes to make sure the docs are ok and have working examples.

I also added the simulate function on the sender, because it was mentionned in the docs ans missing in the code

**Review by CyberHoward — APPROVED** [review]
LGTM!!

**Review by CyberHoward — APPROVED** [review]
LGTM!!

**Pull request merged**